quelpa.el

```diff
@@ -483,8 +486,7 @@ If there is an error and no existing checkout return nil."
            'package-build
            `(:url ,quelpa-melpa-repo-url :files ("*"))
            quelpa-melpa-dir)
-        (error (quelpa-message t "failed to checkout melpa git repo: `%s'" (error-message-string err))
-               (file-exists-p (expand-file-name ".git" quelpa-melpa-dir))))))
```

Comment:
Why did you remove the argument line?

Reply:
it was an ARG for `error`, but there is no format string for `error` so it was not used anyway if I'm not mistaken...

Reply:
Yeah, I'm mostly wondering what the intention was here, it seems like you intended to print both the original error message and the path to `quelpa-melpa-dir`.

Reply:
I thought so too. I figured to display the path as well doesn't make much sense.